### DISCUSSION TITLE
database sqlserver script issue for quartz.net 3.11

### DISCUSSION THREAD

**Discussion by zpjune** [quartznet/quartznet#2496] [Q&A] [answered]
this is an error when I use quartz.net 3.11 in .net 8 webapi, error is Quartz.SchedulerException: 'Database schema validation failed. Make sure you have created the database tables that Quartz requires using the database schema scripts. You can disable this check by setting quartz.jobStore.performSchemaValidation to false , could you please help with it , Thank you very much

**Comment by lahma** [comment, marked as answer]
Did you create the database tables using [the scripts provided](https://github.com/quartznet/quartznet/tree/main/database/tables)?

**Reply by zpjune** [reply]
Hi [lahma], my problem has been fixed ,please ignore it , thanks
